t/01_config/06_config_api.t

```diff
@@ -16,20 +16,20 @@ eval {
     Dancer::Config::load_settings_from_yaml('foo');
 };
 
-like $@, qr/Unable to parse the configuration file/;
+like $@, qr/Unable to parse the configuration file/, 'invalid yaml string';
 
 my $dir = File::Temp::tempdir(CLEANUP => 1, TMPDIR => 1);
 
 my $config_file = File::Spec->catfile($dir, 'settings.yml');
 
 open my $fh, '>', $config_file;
-print $fh '---foo\n';
+print $fh '---"foo\n';
```

Comment:
Given my above comment will force to edit the file, probably remove that \n from there, as it is misleading and unnecessary. (it already way :( )